output/create_kubevirt_yaml.ml

```diff
@@ -66,6 +66,15 @@ let create_kubevirt_yaml source inspect
     );
     List.push_back metadata ("labels", Assoc !labels) in
 
+  (* The target firmware. *)
+  let bootloader =
+    match target_firmware with
+    | TargetBIOS ->
+       "bios", Assoc []
+    | TargetUEFI ->
+       "efi", Assoc ["persistent", Bool true] in
```

Comment:
hmm by default the kubevirt has the `persistent` as `false` but to me keeping the EFI NVRAM across reboots makes sense, not sure why they default to false.

Reply:
It seems a good idea to save NVRAM across boots, especially as v2v-converted guests are pets rather than cattle.